src/modules/core/components/Slider/Slider.tsx

```diff
@@ -0,0 +1,107 @@
+import React, { useState, useMemo, useCallback } from 'react';
+import ReactSlider from 'rc-slider';
+import 'rc-slider/assets/index.css';
+
+type Appearance = {
+  theme?: 'primary' | 'danger';
+};
+
+interface Props {
+  value: number;
+  max?: number;
+  limit?: number;
+  appearance?: Appearance;
+  onChange: (val: any) => void;
+}
+
+const displayName = 'Slider';
+
+const Slider = ({ value, max = 100, onChange, limit, appearance }: Props) => {
+  const [sliderValue, setSliderValue] = useState<number>(value);
+
+  const gradientStopPercentage = useMemo(() => {
+    return limit ? Math.round((limit / max) * 100) : 0;
+  }, [limit, max]);
+
+  const onSliderChange = useCallback(
+    (val): void => {
+      if ((limit && sliderValue < limit) || val < sliderValue || !limit) {
+        setSliderValue(val);
+        onChange(val);
+      }
+      if (limit && sliderValue > limit) {
+        setSliderValue(limit);
+        onChange(limit);
+      }
+    },
+    [setSliderValue, onChange, limit, sliderValue],
+  );
+
+  const marks = {};
+
+  if (limit && limit < max) {
+    marks[limit] = {};
+  }
+
+  const SliderStylesObject = {
+    primary: {
+      backgroundColor: '#68D2D2',
+      borderColor: '#68D2D2',
+      height: 1,
+      markHeight: 5,
+      markWidth: 1,
+      markPositionTop: -2,
+    },
+    danger: {
+      backgroundColor: '#FE5E7C',
+      borderColor: '#FE5E7C',
+      height: 3,
+      markHeight: 8,
+      markWidth: 2,
+      markPositionTop: -3,
+    },
```

Comment:
There's a difference in styles between the two themes of the slider. The danger theme is "thicker" than the primary one:

![Screenshot from 2021-03-09 14-06-33](https://user-images.githubusercontent.com/1193222/110471284-a2f81c00-80e4-11eb-9088-94e034f200e9.png)

Personally I like how the danger one looks, so I would say to make the primary one the same thinckness at the danger theme.

Reply:
haha i actually more like primary more :D But if you wont make them same thickness ask Karol if he is ok with that